api/info.go

```diff
@@ -86,6 +89,28 @@ func (s *InfoService) Get(r *http.Request, args *InfoArgs, reply *InfoReply) err
 		inslog.Error("[ INFO ] rootMember ref is nil")
 		return errors.New("[ INFO ] rootMember ref is nil")
 	}
+	migrationDamonMembers, err := s.runner.GenesisDataProvider.GetMigrationDamonMembers(ctx)
+	if err != nil {
+		inslog.Error(errors.Wrap(err, "[ INFO ] Can't get migration damon members refs"))
```

Comment:
Why not like this?
{code}
msg := "Error text"
inslog.Error( msg )
return msg
{code}
To eliminate doubling of message text